src/startpage/startpage.cpp

```diff
@@ -11,54 +11,65 @@ StartPage::StartPage(QWidget *parent) :
     ui(new Ui::StartPage)
 {    
     ui->setupUi(this);
-    setWindowTitle("Start Page");    
+    setWindowTitle("Start Page");
 
     int maxButtonsWidth = 200;
+    QFont lblFont("Segoe UI", 12);
+
+    //lblFont.setBold(true);
+    QPalette palette;
+    palette.setColor(QPalette::WindowText, Qt::blue);
 
     // creating & configuring new file button
-    pNewBtn = new QPushButton(tr("   New File"));
-    setupButton(pNewBtn, style()->standardIcon(QStyle::SP_FileIcon), maxButtonsWidth);
-    connect(pNewBtn, &QPushButton::pressed, this, &StartPage::newBtnPressed);
+    mpNewBtn = new QPushButton(tr("   New File"));
```

Comment:
For what is this space between " and New?

Reply:
Tried to adjust spacing between icon & text on button.
Fixed that: now icon is aligned to the right & text - to the centre